virocon/distributions.py

```diff
@@ -1347,3 +1347,158 @@ def _fit_mle(self, sample):
 
     def _fit_lsq(self, data, weights):
         raise NotImplementedError()
+ 
+class VonMisesDistribution(Distribution):
+    """
+    A Von Mises (Circular Norm) Distribution. 
+    
+    The distributions probability density function is given by [1]_: 
+    
+    :math:`f(x) = \\frac{\\exp{k \\cos{x - \\mu}}}{2 \\pi I_0(k) \\sigma}`
+     
+    The distribution is used to model wind-wave misalignment in [2]_. 
+    Being a circular norm distribution it can be used to model direction. 
+    
+    Parameters
+    ----------
+    mu : float
```

Comment:
Please use the same order of parameters in the docstring as in the `__init__`. It should usually be shape(s), loc, scale.

Reply:
also noted, thanks again